src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/extensions */
+
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const operandOne = Big(numberOne);
+  const operandTwo = Big(numberTwo);
+
+  switch (operation) {
+    case 'X':
+      return operandOne.times(operandTwo).toString();
+    case '-':
+      return operandOne.minus(operandTwo).toString();
+    case '+':
+      return operandOne.plus(operandTwo).toString();
+    case '%':
+      return operandTwo.div('100').toString();
+    case '÷':
+      if (operandTwo === 0) return 0;
+      return operandOne.div(operandTwo).toString();
```

Comment:
Manage division by ZERO in milestone 5, we'll verify that. It shouldn't return `0`, but something else. something like `NaN` or `INFINITY`....or some other creative error management you'll think about.

### Correct this in milestone 5